src/main/java/ch/admin/bag/covidcertificate/gateway/service/dto/incoming/CovidCertificatePersonDto.java

```diff
@@ -9,6 +9,6 @@
 @AllArgsConstructor
 public class CovidCertificatePersonDto {
     private CovidCertificatePersonNameDto name;
-    @Schema(example= "1964-03-14", description = "birthdate of the covid certificate owner. Format: ISO 8601 date without time.")
+    @Schema(example= "1985-09-20, 1985-09, 1985", description = "birthdate of the covid certificate owner. Format: ISO 8601 date without time OR YYYY-MM OR YYYY.")
```

Comment:
Wenn `example` so angepasst wird, dann ist das automatisch generierte JSON nun ungültig (da ja nur einer der drei Beispiel-Werte gleichzeitig verwendet werden darf). Evt. wäre es besser, nur die `description` anzupassen?

![image](https://user-images.githubusercontent.com/49515902/147910102-2ad67e64-ab72-4b68-a3d8-7a6cbc2d0938.png)

```suggestion
    @Schema(example= "1985-09-20", description = "birthdate of the covid certificate owner. Format: ISO 8601 date without time OR YYYY-MM OR YYYY.")
```